get-config-state.js

```diff
@@ -40,6 +40,10 @@ function getConfigState(dirname, opts) {
         dc && NODE_ENV ?
             join(configFolder, NODE_ENV + '.' + dc.datacenter + '.json') :
             null,
+        // load ./config/secrets-NODE_ENV.json
+        NODE_ENV ? join(configFolder, 'secrets-' + NODE_ENV + '.json') : null,
```

Comment:
do we actually need to support secrets-NODE_ENV ?

Shouldnt there only be one secret at a time ? Since we symlink it in I dont think we need to put NODE_ENV in the title.